doc/mediasdk-man.md

```diff
@@ -7564,6 +7564,37 @@ This structure is used to pass decryption status report index for Common Encrypt
 
 This structure is available since SDK API 1.30.
 
+## <a id='mfxExtPartialBitstreamParam'>mfxExtPartialBitstreamParam</a>
+
+**Definition**
+
+```C
+typedef struct {
+    mfxExtBuffer    Header;
+    mfxU32          BlockSize;        /* output block granulatiry for Granularity = MFX_PARTIAL_BITSTREAM_BLOCK */
+    mfxU16          Granularity;      /* granulatiry of the partial bitstream: slice/block/any */
+    mfxU16          reserved[8];
+} mfxExtPartialBitstreamParam;
+```
+
+**Description**
+
+This structure is used by AVC HW encoder to output parts of bitstream as soon as they ready. The application can attach this extended buffer to the [mfxFrameSurface1](#mfxFrameSurface) structure at init time.
+If this option is turned ON and session configured in single thread, then AVC encoder can output bitstream by part based with required granularity. <br>
```

Comment:
> option is turned ON

In this context (descring ext buffer structure) is confusing. What means "turned ON" - when buffer is attached to frame surface or anything else?

> session configured in single thread

What do you mean here? AsyncDepth=1 or mediasdk library initialization via MFXInitEx with threads=1?

Reply:
"turned ON" means Granularity !=MFX_PARTIAL_BITSTREAM_NONE and buffer is attached
"session configured in single thread" means mfxIMPL in the session initialization should contain MFX_IMPL_EXTERNAL_THREADING

Reply:
I would add in the man: "If this option is turned ON, i.e. mfxExtPartialBitstreamParam is attached and Granularity !=MFX_PARTIAL_BITSTREAM_NONE, ..." to make it clear.

I don't see MFX_IMPL_EXTERNAL_THREADING description in mediasdk manual. What is that?